### ISSUE TITLE
Vertical Scroll Not visible after scrolling web page off Handsontable then back on

### ISSUE THREAD

**Issue by nouryf** [handsontable/handsontable#3723]
Hi,

I created a simple handsontable and fill with data:
	myGrid = new Handsontable(container, {
	   data: data,
	    colHeaders: true,
	    rowHeaders: true,
	    scrollV: 'auto',
	    height: 200,
	});

...

When displaying on my web page, the table has vertical scroll bar displaying Ok. I scrolled my web page down to see other web page contents (this make my handsontable not visible any more). Then I scrolled my web page up to look at my handsontable: Vertical Scroll bar is not there any more !! White space replaced it and now I can't vertically scroll my rows... (See attached screenshots). The screenshot with green rectangle in it is the table when displayed the first time. The second one (red rectangle at right side) is the table after being scrolled off the web page then back on.

I'm using version  0.20.0. I have this issue in both FireFox and Chrome.

I noticed that i needed to click on the extra white space (replacing the v scroll bar) few times and some times I get back the v scroll bar but not all the times.

Is this is a known issue ? Or is there a way to make the vertical scroll not go away ?

Noury

Thanks

**Comment by nouryf** [comment]
![verticalscroll_when_displayed](https://cloud.githubusercontent.com/assets/18246160/17873489/c3abdd9c-6893-11e6-9496-6aa838be1493.png)
![verticalscroll_after_scrolled_off_and_back_on](https://cloud.githubusercontent.com/assets/18246160/17873490/c3ac0038-6893-11e6-8a76-57baf7a4e3c9.png)

**Comment by nouryf** [comment]
![verticalscroll_after_scrolled_off_and_back_on](https://cloud.githubusercontent.com/assets/18246160/17873524/043971ee-6894-11e6-9bcf-58971bbc6cfa.png)
![verticalscroll_when_displayed](https://cloud.githubusercontent.com/assets/18246160/17873525/04399804-6894-11e6-92a6-86ca2ab9b4bb.png)

**Comment by nouryf** [comment]
![verticalscroll_after_scrolled_off_and_back_on](https://cloud.githubusercontent.com/assets/18246160/17873552/300bd816-6894-11e6-9ec7-a56728eb9eb3.png)
![verticalscroll_when_displayed](https://cloud.githubusercontent.com/assets/18246160/17873553/3010865e-6894-11e6-9fb7-61db8267b7d1.png)

**Comment by nouryf** [comment]
![verticalscroll_after_scrolled_off_and_back_on](https://cloud.githubusercontent.com/assets/18246160/17873567/3f37febe-6894-11e6-98c2-6084e42b53fc.png)
![verticalscroll_when_displayed](https://cloud.githubusercontent.com/assets/18246160/17873571/425e9f94-6894-11e6-970d-552c893c6ba0.png)

**Comment by AMBudnik** [comment]
Hi @nouryf 
sorry for the silly question but by writing the `scrollV` you mean that you've added the `scrollViewportTo`?
It would be very helpful if you could create a demo (for example in the JSFiddle).

**Comment by nouryf** [comment]
Hi @AMBudnik,

Thanks for your response. The 'scrollV' is I believe a property of the table to state 'vertical scroll' needs to be on 'auto' mode. I did not add a scrollviewportto.

While I was trying to trim the code and have a simple reproducible demo, I figured out what was the problem and found a solution to it. I'll explain and you let me know if you still need a demo for the issue or not. 

Basically the web page when it load the first time it does not have a vertical scroll bar as there is only the handsontable in it with sample data. The handsontable has its vertical scroll bar visible and working fine.

Now once I click on a button in the web page, this make the web page longer as I show some hidden content at the bottom and now suddenly the web page have a vertical scroll bar. That on the fly vertical bar affect the scroll bar of the hands on table as it hide its or its being drawn on the top of it. Basically the handsontable vertical scroll bar is now in bad state. My fix, is simply to update the handsontable with : 	            
myGrid.updateSettings({
	            	scrollV: 'auto',
	            	stretchH: "all"
	            });

and this 'refresh' the layout of the handsontable making its vertical scroll bar visible again.

If you think this behavior is an issue in Handsontable not refreshing on the fly when the page change, let me know and I'll send the reproducible demo. Otherwise I would consider a user error.

Thanks
